src/wp-includes/media.php

```diff
@@ -925,6 +929,10 @@ function wp_get_registered_image_subsizes() {
 			$size_data['crop'] = (bool) $size_data['crop'];
 		}
 
+		if ( isset( $additional_sizes[ $size_name ]['output_mimes'] ) ) {
+			$size_data['output_mimes'] = $additional_sizes[ $size_name ]['output_mimes'];
+		}
```

Comment:
I'm not sure we need to include this here, since it's not really relevant when using the sizes. The `_wp_filter_image_sizes_additional_mime_type_support()` should be used to check for a list of sizes whether they have this flag set or not.

The problem of including this here is also that then it makes the data inconsistent, since core sizes now don't have the new flag set.

Reply:
removed